src/ripple/app/tx/impl/OfferStream.h

```diff
@@ -174,6 +174,16 @@ class FlowOfferStream : public TOfferStreamBase<TIn, TOut>
 public:
     using TOfferStreamBase<TIn, TOut>::TOfferStreamBase;
 
+    // The following interface allows offer crossing to permanently
+    // remove self crossed offers.  The motivation is somewhat
+    // unintuitive.  See the discussion in the testSelfCrossOffer1()
+    // unit test.
+    void
+    permRmOffer (uint256 const& offerIndex)
```

Comment:
This does exactly what the `sle` version of the `permRmOffer` does. What about making `permRmOffer` public and re-using that function?